lib/index.js

```diff
@@ -25,15 +27,28 @@ function initServer() {
     return server;
 }
 
-function initLogger() {
+function initLogger(abbreviation) {
     const bunyan = require('bunyan');
+    // initialize maximum body logging length
+    bodyAbbreviationLength = abbreviation;
     return bunyan.createLogger({
         name: packageJson.name
     });
 }
 
 function logJson(logger, body) {
-    logger.info(JSON.stringify(body, null, 4));
+    // Make a copy so the original body doesn't get mutated
+    const content = Object.assign({}, body);
+    // If the body is too long of a string, abbreviate it
+    if (bodyAbbreviationLength > 10 && content.body && content.body.length && content.body.length > bodyAbbreviationLength) {
```

Comment:
Why 10?  That would mean `--abbrev 5` wouldn't abbreviate at all, right?  Seems like the last comparison (`length` vs the limit) is the only one we need.